api/v1alpha1/selfnoderemediationconfig_webhook.go

```diff
@@ -95,15 +96,19 @@ var _ webhook.Validator = &SelfNodeRemediationConfig{}
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type
 func (r *SelfNodeRemediationConfig) ValidateCreate() error {
 	selfNodeRemediationConfigLog.Info("validate create", "name", r.Name)
-
-	return r.validateTimes()
+	if err := r.validateTimes(); err != nil {
```

Comment:
Better run all validations ad aggregate errors, so users know about all problems in one run.
See e.g. https://github.com/medik8s/node-healthcheck-operator/blob/main/api/v1alpha1/nodehealthcheck_webhook.go#L116